website/src/components/CopyBoxCommand.tsx

```diff
@@ -1,34 +1,26 @@
+import { useTheme } from '@emotion/react'
 import styled from '@emotion/styled'
-import { CopyButton, Tabs } from '@scaleway/ui'
-import {
-  Children,
-  ReactElement,
-  cloneElement,
-  isValidElement,
-  useState,
-} from 'react'
+import { Stack, Tabs } from '@scaleway/ui'
+import { Children, ReactElement, isValidElement, useState } from 'react'
 import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter'
-import { darcula } from 'react-syntax-highlighter/dist/esm/styles/prism'
+import {
+  dracula,
+  oneLight,
+} from 'react-syntax-highlighter/dist/esm/styles/prism'
+
+const StyledDiv = styled(Stack)`
```

Comment:
```suggestion
const StyledStack = styled(Stack)`
```

no ?

Reply:
@chambo-e yes you are right I'm changing it